src/lib/modules/SearchControls/SearchError.js

```diff
@@ -1,21 +1,48 @@
+import React, { Component } from 'react';
 import { Header, Icon, Segment } from 'semantic-ui-react';
-import React from 'react';
 import PropTypes from 'prop-types';
+import _get from 'lodash/get';
 
-export const SearchError = ({ error }) => {
-  return (
-    <Segment placeholder textAlign="center">
-      <Header icon>
-        <Icon name="warning" />
-        Oups! Something went wrong while fetching results.
-      </Header>
-      <Header.Subheader>
-        Please try again later. If the problem persists, contact the technical
-        support.
-      </Header.Subheader>
-    </Segment>
-  );
-};
+export class SearchError extends Component {
+  renderInvalidQueryError = () => {
+    return (
+      <Segment placeholder textAlign="center">
+        <Header icon>
+          <Icon name="question" />
+          Unrecognized search query.
+        </Header>
+        <Header.Subheader>
+          The query you entered was not recognized. Please refer to the
+          documentation for more information.
```

Comment:
Can we remove the part `refer to the documentation`? We still have to create such doc! :)
There is an issue to create advanced searches, but it is not yet done...

```suggestion
          title: Search query error.
          msg: The query is malformed or contains invalid characters.
```

Reply:
OK, I was thinking about that too; will change, thanks.